### PR TITLE
fix: soap example

### PR DESCRIPTION
**Pull request by nabdelgadir**
* If using lb4 example to clone soap example, the calculator ds json file gets created

Using lb4 example for the soap calculator would not produce the calculator.datasource.json file in dist8 so the example would not run. This PR is created to fix the issue. 

## Checklist

- [x] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [x] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

### CODE REVIEW

File: examples/soap-calculator/index.d.ts [hunk 1]
```diff
@@ -0,0 +1,5 @@
+// Copyright IBM Corp. 2018. All Rights Reserved.
+// Node module: @loopback/example-todo-list
```
Comment (virkt25) [inline examples/soap-calculator/index.d.ts]: The module name should be updated to `@loopback/example-soap-calculator`

File: examples/soap-calculator/tsconfig.build.json [hunk 2]
```diff
@@ -1,14 +1,12 @@
 {
   "$schema": "http://json.schemastore.org/tsconfig",
   "extends": "./node_modules/@loopback/build/config/tsconfig.common.json",
+  "compilerOptions": {
+    "rootDir": "."
+  },
   "include": [
+    "index.ts",
```
Comment (virkt25) [inline examples/soap-calculator/tsconfig.build.json]: nitpick: the 3 entries of this array can be on the same line.

**Comment by nabdelgadir** [comment]
> Is the change to .vscode/settings.json intentional?

No, it wasn't intentional, just changed it back.

File: .vscode/settings.json [hunk 3]
```diff
@@ -23,7 +23,6 @@
   },
   "files.insertFinalNewline": true,
   "files.trimTrailingWhitespace": true,
-
```
Comment (nabdelgadir) [inline .vscode/settings.json]: Is the empty line important/necessary here?
It's removed by autosave, but if yes, I can revert the change.

Comment (virkt25) [reply .vscode/settings.json]: No it's not necessary. Probably best to remove it.

File: examples/soap-calculator/package.json [hunk 4]
```diff
@@ -43,14 +43,6 @@
...
   },
   "license": "MIT",
-  "files": [
-    "README.md",
-    "index.js",
-    "index.d.ts",
-    "dist*/src",
-    "dist*/index*",
-    "src"
-  ],
```
Comment (raymondfeng) [inline examples/soap-calculator/package.json]: Have we decided to remove the explicit listing of `files` in `package.json` now? IIRC, it was there to exclude certain files to be published. 

If so, we should clean up all packages.

Comment (virkt25) [reply examples/soap-calculator/package.json]: None of the other example packages have a `files` field.

**Comment by dhmlau** [comment]
@nabdelgadir, thanks for your first contribution to LB!

**Pull request merged**